examples/close_cookies/close_cookies.py

```diff
@@ -0,0 +1,45 @@
+"""This example demonstrates how to close popup windows (like promotion form) with AgentQL."""
+
+import time
+
+from agentql.ext.playwright.sync_api import Page
+from playwright.sync_api import sync_playwright
+
+# Set the URL to the desired website
+URL = "https://www.amazon.com/"
+
+QUERY = """
+{
+    cookies_form {
+        reject_btn
+    }
+}
+"""
+
+
+def main():
+    with sync_playwright() as playwright:
+        browser = playwright.chromium.launch(headless=False)
+
+        # Create a new AgentQL page instance in the browser for web interactions
```

Comment:
I like this more, we don't exactly create an "AgentQL page"
```suggestion
        # Create a new page in the browser and cast it to custom Page type to get access to the AgentQL's querying API
```